forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -363,3 +364,68 @@ def get():
             )
         except BusinessException as err:
             return err.error, err.status_code
+
+
+@cors_preflight("POST,OPTIONS")
+@API.route("/create-application", methods=["POST", "OPTIONS"])
+class ApplicationCreation(Resource):
+    """Resource for application creation."""
+
+    @staticmethod
+    @auth.require
+    @profiletime
+    def post():
+        """Post a new application using the request body.
+
+        : data: form submission data
+        : formId:- Unique Id for the corresponding form
+        """
+        formio_url = current_app.config.get("FORMIO_URL")
+        web_url = current_app.config.get("WEB_BASE_URL")
+        application_json = request.get_json()
+        data = request.get_json()
+        try:
+            application_schema = ApplicationSchema()
+            dict_data = application_schema.load(application_json)
```

Comment:
Please provide meaningful name instead of dict_data, like is it application_data ?